build.gradle

```diff
@@ -19,7 +19,7 @@ plugins {
 }
 
 
-version = '2.2.5'
+version = '2.2.6'
```

Comment:
I think this is a new feature more than a bug fix, so maybe we could do `2.3.0` ?